appveyor.yml

```diff
@@ -2,23 +2,26 @@ image:  Ubuntu1804
 
 install:
   - |
-    chef_version='0.9.42'
+    chef_version='21.5.462'
```

Comment:
I would like to upgrade this, but this is the last version that uses Ruby 2.7. Ruby 3.0 is trying to execute an illegal instruction as soon as test kitchen starts. I am not sure if upgrading the Ubuntu env would help or not; it seems like a hardware support issue but maybe the platform would matter?